.circleci/config.yml

```diff
@@ -109,17 +114,47 @@ jobs:
     docker:
       - image: cimg/node:18.16.0
     steps:
-      - install_dependencies
-      - run: echo "!TODO"
+      - add_ssh_keys:
+          fingerprints:
+            - '97:69:fe:70:ca:84:59:06:98:ab:9c:ec:d5:26:49:8c'
+      - checkout
+      - setup_remote_docker:
+          version: 20.10.14
+      - run:
+          name: Build App image
+          command: docker build -t kuba1212/recipeapp-docker .
+      - run:
+          name: Save built image
+          command: docker save kuba1212/recipeapp-docker | gzip > appimage.tar.gz
+      - create_env
+      - run:
+          name: Send production env
+          command: |
+            scp -o StrictHostKeyChecking=accept-new ./.env $SSH_USER@$SSH_HOST:.env
+      - run:
+          name: Send image to server
+          command: |
+            scp -o StrictHostKeyChecking=accept-new ./appimage.tar.gz $SSH_USER@$SSH_HOST:appimage.tar.gz
+      - run:
+          name: Send docker compose file
+          command: |
+            scp -o StrictHostKeyChecking=accept-new ./docker-compose.yml $SSH_USER@$SSH_HOST:docker-compose.yml
+      - run:
+          name: Deploy over SSH
+          command: |
+            ssh -o StrictHostKeyChecking=accept-new $SSH_USER@$SSH_HOST "source .env && ./recipe_app_setup.sh"
 
 workflows:
   build_and_test:
     jobs:
-      - build
+      - build:
+          context: test
```

Comment:
If you need this context, name it in a descriptive way so that we know what it's responsible for

Reply:
Take a note that context is shared across all CI projects on the account. So it should at minimum include a project name.